tests/HandlerTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace ReadMe\Tests;
+
+class HandlerTest extends \PHPUnit\Framework\TestCase
+{
+    public function testHandlerImplementsCoreHandler(): void
+    {
+        require_once(__DIR__ . '/../src/handler.dist.php');
+
+        $handler = new \App\Handler\ReadMe;
+
+        $this->assertTrue($handler instanceof \ReadMe\Handler);
```

Comment:
I remember reading somewhere that you should use the most specific assertion possible. This will generate create a more readable error message. I haven't run this code, but my assumption would be _expected true and got false_ compared to _Expected instance of ReadMe\Handler and got Foo\FizzBar_. tl;dr you might try `assertInstanceOf($handler, Handler::class)`